DQM/HLTEvF/plugins/TrigObjTnPHistColl.cc

```diff
@@ -0,0 +1,399 @@
+#include "TrigObjTnPHistColl.h"
+
+#include "FWCore/Common/interface/TriggerNames.h"
+
+namespace{
+  std::vector<float> convertToFloat(const std::vector<double>& vecD){
+    std::vector<float> vecF;
+    for(double x : vecD) vecF.push_back(x);
+    return vecF;
+  }
+}
+
+TrigObjTnPHistColl::TrigObjTnPHistColl(const edm::ParameterSet& config):
+  tagCuts_(config.getParameter<std::vector<edm::ParameterSet>>("tagCuts")),
+  probeCuts_(config.getParameter<std::vector<edm::ParameterSet>>("probeCuts")),
+  tagFilters_(config.getParameter<edm::ParameterSet>("tagFilters")),
+  collName_(config.getParameter<std::string>("collName")),
+  folderName_(config.getParameter<std::string>("folderName")),
+  histDefs_(config.getParameter<edm::ParameterSet>("histDefs")),
+  evtTrigSel_(config.getParameter<edm::ParameterSet>("evtTrigSel"))
+  
+{
+  for(auto probeFilter : config.getParameter<std::vector<std::string> >("probeFilters")){
```

Comment:
`auto & probeFilter` should avoid a copy (`getParameter` returns by value anyway)

Reply:
fixed, thanks